app/scripts/lib/createOnboardingMiddleware.js

```diff
@@ -3,18 +3,18 @@ import extension from 'extensionizer'
 
 /**
  * Returns a middleware that intercepts `wallet_registerOnboarding` messages
- * @param {{ location: string, tabId: number, registerOnboarding: Function }} opts - The middleware options
+ * @param {{ location: string, registerOnboarding: Function }} opts - The middleware options
  * @returns {(req: any, res: any, next: Function, end: Function) => void}
  */
-function createOnboardingMiddleware ({ location, tabId, registerOnboarding }) {
```

Comment:
Do we need to update the call sites?

Reply:
Yes... somehow that didn't get committed 🤔 I could have sworn I did that, whoops.